ts/ts/main.ts

```diff
@@ -0,0 +1,375 @@
+import * as THREE from '../js/three.module.js';
+import {
+  TextureLoader,
+  Scene,
+  WebGLRenderer,
+  Vector3,
+} from '../js/three.module.js';
+import { GLTFLoader } from '../js/GLTFLoader.js';
+
+const initialDelayBeforeCalendarPageFlip = 1000; // in ms
+const breakpointForLarge = 1024;
+
+const currentDate = new Date();
+// const currentYear = currentDate.getFullYear();
+const currentMonth = currentDate.getMonth(); // this is 0-indexed
+const currentDay = currentDate.getDate();
+const previousDate = new Date();
+previousDate.setDate(previousDate.getDate() - 1);
+const previousMonth = previousDate.getMonth(); // this is 0-indexed
```

Comment:
Definitely remove the commented out code on line 14. Debatable remove the other commented out code if it's not necessary too, or make it into a more fleshed out comment such as:
`// getMonth returns a 0-indexes something`